lib/version_utils.pm

```diff
@@ -194,6 +195,16 @@ sub check_version {
     croak "Unsupported version parameter for check_version: '$query'";
 }
 
+=head2 is_microos
+
+Check if product is SUSE MicroOS
+Version: 5.X, 6.X
+=cut
+sub is_microos {
+    return get_var('DISTRI') =~ /microos/ && get_var('VERSION') =~ "^[56].";
```

Comment:
```suggestion
    return get_required_var('DISTRI') =~ /microos/ && get_required_var('VERSION') =~ "^[56]\.";
```
I guess you wanna match the `.` and not any character...

Reply:
> I guess you wanna match the . and not any character...
Ok, thanks.

I'd like to keep `get_var` as the entire .pm uses get_var in all the cases. Either we change it everywhere or keep the same everywhere. `get_required_var` is not even loaded in this pm...